slycot/src/transform.pyf

```diff
@@ -528,3 +528,40 @@ subroutine tg01fd_uu(compq,compz,joba,l,n,m,p,a,lda,e,lde,b,ldb,c,ldc,q,ldq,z,ld
 integer required intent(in) :: ldwork
     integer intent(out) :: info
 end subroutine tg01fd_uu
+subroutine mb03rd_n(jobx,sort,n,pmax,a,lda,x,ldx,nblcks,blsize,wr,wi,tol,dwork,info) ! in MB03RD.f
```

Comment:
It may be just my personal opinion, but I don't think it is a good idea to introduce a specific wrapper for different job options with different parameter and return signatures when there is no use case and no unit test for it. Why not just keep one wrapper  for the function, test that with the default `job='U'` and deal with 'N' as soon as it becomes relevant?